src/style/CharityPage.less

```diff
@@ -66,47 +65,43 @@
 			}
 		}
 	} // ./column
-
-	.top-bands {
-		position: fixed;
-		top: @navbar-height;
-		left: 0;
-		width: 100%;
-		.band1 {
-			background: linear-gradient(to right, #819EB7, #C1DEF2);
-			height: @band-1-height;
-		}
-		.band2 {
-			background: linear-gradient(to right,#ACB09F, #D2D6C3);
-			height: @band-2-height;
-		}
-		.band3 {
-			background: linear-gradient(to right, #819EB7, #C1DEF2);
-			height: @band-3-height;
-		}
-	}
 }
 
 #charity {
 	.impact-column {
 		.header {
-			background: #C1DEF2;
 			.extend-bg-to-sides(@bs-column-padding);
 			padding-top: @band-1-height;
 			margin-bottom: 30px;
 
 			.charity-name {
-				background-color: #D2D6C3;
-				text-transform: uppercase;
-				font-weight: bold;
+				text-transform: capitalize;
+				font-family: 'Nunito Sans';
```

Comment:
Let's create a fonts.less file to hold font definitions. 

Also, to promote consistent text styling and easier to maintain css -- let's define a few header styles there, then we can use those instead of a specific charity-name header style, and  so on.

The css in these files isn't the best organised -- this is a good opportunity to clean out some of the old, and replace with better structured easier-to-maintain css.

Thanks!

Reply:
Done